drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/testutil/TestUtil.java

```diff
@@ -85,10 +86,25 @@ public static void assertContains(final String expected,
                                       final Set<Integer> rowNumbers,
                                       final Set<Issue> result) {
 
-        assertTrue("Could not find: " + expected + " for rows: " + rowNumbers,
-                   result.stream().filter(issue -> titleEquals(issue,
-                                                               expected) && rowNumberContains(issue,
-                                                                                                rowNumbers)).count() > 0);
+        assertContains(expected,
+                       rowNumbers,
+                       Severity.WARNING,
+                       result);
+    }
+
+    public static void assertContains(final String expected,
+                                      final Set<Integer> rowNumbers,
+                                      final Severity severity,
+                                      final Set<Issue> result) {
+
+        assertTrue("Could not find: " + expected + " for rows: " + rowNumbers + " and severity: " + severity,
+                   result.stream()
+                         .filter(issue -> issue.getSeverity() == severity
+                                          && titleEquals(issue,
+                                                         expected)
+                                          && rowNumberContains(issue,
+                                                               rowNumbers))
+                         .count() > 0);
```

Comment:
I decided not to apply, because I think you can't call `empty()` or `isEmpty()` or something like that directly on stream, you have to collect elements at first. I think `Stream.filter().count() == 0` is not so much worse than `Stream.filter().collect().isEmpty()`